app/manifest.webapp

```diff
@@ -1,14 +1,14 @@
 {
-  "version": "0.1.0",
+  "version": "1.0.0",
   "name": "CohortBuilder",
   "description": "modern refresh of the OpenMRS Cohort Builder tool",
   "launch_path": "index.html",
   "icons": {
     "48": "/img/omrs-button.png"
   },
   "developer": {
-    "name": "andela-odaniel",
-    "url": "https://github.com/andela-odaniel/openmrs-module-cohortbuilder"
```

Comment:
Shouldn't we just add names instead of replacing the original authors?
Also shouldn't the repository url be https://github.com/openmrs/openmrs-owa-cohortbuilder